src/src/targets/EWRF_E7082VM/EWRF_E7082VM.c

```diff
@@ -10,7 +10,17 @@
 
 #define OUTPUT_POWER_INTERVAL 5 // ms
 
-uint8_t powerLevels[] = {0, 1, 14, 20, 26}; // SA2.1 powerlevels dBm. Currently only used for Inav and a max of 5.
+/* SA2.1 powerlevels in dBm.
+ *
+ * INav:
+ *    Max of 5 [https://github.com/iNavFlight/inav/blob/a8016edd0d6f05bb12a75b0ea75a3483772baaeb/src/main/io/vtx_smartaudio.h#L36]
+ *    Index 0 is ignored [https://github.com/iNavFlight/inav/blob/a8016edd0d6f05bb12a75b0ea75a3483772baaeb/src/main/io/vtx_smartaudio.c#L334]
+ *
+ * ==> Store RACE_MODE value into index 0 for other FC firmwares.
+ *     RACE_MODE sets VTX into pitmode as 0 does as well but forces
+ *     output power to 14dBm when pitmode is disabled.
+ */
+uint8_t saPowerLevelsLut[SA_NUM_POWER_LEVELS] = {RACE_MODE, 1, 14, 20, 26};
```

Comment:
"1" should be first in the array for Low power on Disarm.

Reply:
reverted.